tests/functional/adapter/test_formats.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2022 Cloudera Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from dbt.tests.util import (
+    run_dbt,
+    check_result_nodes_by_name,
+    relation_from_name,
+    check_relation_types
+)
+
+from tests.functional.adapter.test_basic import (
+    TestSimpleMaterializationsImpala,
+    TestIncrementalImpala
+)
+
+from dbt.tests.adapter.basic.files import (
+    seeds_base_csv,
+    base_view_sql,
+    schema_base_yml
+)
+
+def isIcebergTable(project, tableName):
```

Comment:
cool lets follow the python function naming convention ie function names should be lower case and using underscore. Ideally we should have the pip8 lint checks